### PR TITLE
Add messageTypesToIgnoreInPanel config

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes https://github.com/steelbrain/linter-ui-default/issues/43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,14 @@
         "Right"
       ],
       "default": "Right"
+    },
+    "messageTypesToIgnoreInPanel": {
+      "description": "If you want to see a message type higlighted in editor but not in panel",
```
Comment (Arcanemagus) [inline package.json]: "Comma separated list of message types to show _only_ in the editor, and not in the panel."
(Not sure if that italicization will show in the settings view, cut it out if not.)

Comment (steelbrain) [inline package.json]: GFM seems to work, updated

**Comment by Arcanemagus** [comment]
Specs?

**Comment by steelbrain** [comment]
We are defering the specs for this package and will add them all at once, see #22
But it's not super important, I guess we could make this PR the first one to have them

**Comment by Arcanemagus** [comment]
I'd say it's a much better idea to write them as you go, that way:

* One won't get forgotten
* It won't be as big of a chore to get them all written at once

**Comment by Arcanemagus** [comment]
Back when #22 was filed this was _just_ beginning, that was meant as a reminder that you should have specs at the start there, not that they should eventually be added :stuck_out_tongue:.

**Comment by Arcanemagus** [comment]
LGTM other than specs btw.

**Pull request merged**